insights-client.spec

```diff
@@ -43,6 +43,16 @@ BuildRequires: systemd-rpm-macros
 %description
 Sends insightful information to Red Hat for automated analysis
 
+%package ros
+Requires: pcp-zeroconf
+Summary: The subpackage for Resource Optimization Service
+
+%description ros
+
+The ros subpackage add ros_collect configuration parameter to insights-client.conf file,
+the parameter is set to True by default. The system starts sending PCP archives to
+Resource Optimization service upon modifying ros_collect parameter to True.
```

Comment:
This isn't accurate. The subpackage does not add a configuration value to insights-client.conf; it is present in the config file even if this subpackage is not installed.

Reply:
Agree, updating it

Reply:
I kept this as is as now it adds the param.